TaskWithStrlen/TaskWithStrlen.cpp

```diff
@@ -0,0 +1,34 @@
+﻿#include <iostream>
+
+int strLength(char str[]) {
```

Comment:
Consider declaring the parameter for the function as const char*. Good manners are to declare variables as a const if your code doesn't change it. Indeed we iterate through the array but we don't change anything withing. Good chance to use the const keyword.

Reply:
Fixed